Guardian/Authentication.swift

```diff
@@ -136,6 +160,15 @@ struct RSAAuthentication: Authentication {
             return self.api.resolve(transaction: transactionToken, withChallengeResponse: jwt)
         }
     }
+
+    func handleAction(withIdentifier identifier: String, andNotification notification: Notification) -> VoidRequest {
+        switch identifier {
+        case acceptActionIdentifier:
+            return allow(notification: notification)
+        default:
+            return reject(notification: notification)
```

Comment:
reject by default, doesn't matter if the identifier matches exactly. 
or do we want to throw/return an error here?